FastSimulation/Calorimetry/python/Calorimetry_cff.py

```diff
@@ -262,7 +262,7 @@
             timeShiftHF = cms.vdouble(50.7, 52.5, 52.9, 53.9, 54.5, 55.1, 55.1, 55.7, 55.9, 56.1, 56.1, 56.1, 56.5),
             ),
         HFShower           = cms.PSet(
-            ProbMax          = cms.double(1.0),
+            ProbMax          = cms.double(0.5),
```

Comment:
this is not ideal.
the ProbMax should stay 1.0 for Run1 and then change to 0.5 from the Run2 and beyond.
https://github.com/cms-sw/cmssw/pull/25155/files#diff-5ce5dd838a17416becd944c6910f93ea61928b0bc0ed63ddfb9b8ad19347d974R534 

Better import directly the parameter from FullSim config

Reply:
Ok thanks for catching this and providing the solution.